impl/src/planning.rs

```diff
@@ -416,82 +496,122 @@ impl<'planner> WorkspaceSubplanner<'planner> {
 
   /** Produces a crate context for each declared crate and dependency. */
   fn produce_crate_contexts(&self) -> Result<Vec<CrateContext>> {
+    // Build a list of all catalogs we want the context for
+    let mut catalogs = vec![self.crate_catalog];
+    for catalog in self.binary_dependencies.iter() {
+      catalogs.push(catalog);
+    }
+
+    // Build a list of all workspace files
+    let mut files: Vec<&CargoWorkspaceFiles> =
+      self.binary_deps_files
+      .iter()
```

Comment:
Indentation is weird here. Run rustfmt?

(be ready to revert if it causes massive diffs in unchanged code)

Reply:
I did run `rustfmt`.  Perhaps the config needs to be updated?